src/panels/CreateClusterPanel.ts

```diff
@@ -162,11 +164,32 @@ async function createCluster(
         errorMessage: null,
     });
 
-    const clusterSpec = getManagedClusterSpec(location, name);
+    // kubernetes version is required to create a cluster via deployments
+    const kubernetesVersion = await containerServiceClient.managedClusters.listKubernetesVersions(location);
+    if (!kubernetesVersion || !kubernetesVersion.values || kubernetesVersion.values.length === 0 || !kubernetesVersion.values[0].version) {
```

Comment:
Thanks heaps. just a note: this `isDefault` if we can add note or comment in this PR for doc reference, in case this is changed in coming future et. al. we don't need to search olde reference, while we already know where the docs reside.

Reply:
Updated the PR.